common/djangoapps/student/views/dashboard.py

```diff
@@ -629,6 +632,21 @@ def student_dashboard(request):
 
     enterprise_message = get_dashboard_consent_notification(request, user, course_enrollments)
 
+    recovery_email_message = None
+
+    # Update the tag info with 'enterprise_learner' if the user belongs to an enterprise customer.
+    is_enterprise_learner_data = is_enterprise_learner(user=request.user)
+    does_secondary_email_exist = AccountRecovery.objects.filter(user=user).values_list('secondary_email')
+    if is_enterprise_learner_data and is_secondary_email_feature_enabled() and not does_secondary_email_exist:
```

Comment:
You can use [`is_secondary_email_feature_enabled_for_user `](https://github.com/edx/edx-platform/blob/master/openedx/core/djangoapps/user_api/accounts/utils.py#L207) here instead.
```
if is_secondary_email_feature_enabled_for_user(request.user) and not does_secondary_email_exist:
    ...
```